bin/upload_buildkite_test_analytics_junit

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -eu
+
+FILE_PATH=$1
+ANALYTICS_TOKEN=$2
+
+if [ -z "$FILE_PATH" ]; then
+	echo "You must pass a path to the file you want to upload to Test Analytics"
+	exit 1
+fi
+
+if [ -z "$ANALYTICS_TOKEN" ]; then
+	echo "You must pass a Buildkite Analytics Token"
+	exit 1
+fi
+
+if [ ! -f "$FILE_PATH" ]; then
+	echo "No file exists at $FILE_PATH. Cancelling Upload"
+	exit 0
+fi
+
+echo "Uploading ${FILE_PATH} to Test Analytics"
+
+curl -X POST \
+  -H "Authorization: Token token=$ANALYTICS_TOKEN" \
```

Comment:
Have you thought of calling this `BUILDKITE_ANALYTICS_TOKEN`? 

We're the ones who set the value, and I can see how it might be good to avoid the `BUILDKITE_` prefix to differentiate from the other env vars that come from the agent. On the other hand, all the documentation I interacted with so far, and the source of the Test Analytics Swift library all refer to this as `BUILDKITE_ANALYTICS_TOKEN` 🤷‍♂️

Reply:
I did, but I actually avoided it intentionally so that if `BUILDKITE_ANALYTICS_TOKEN` was set, it wouldn't just automatically be used. This is because some projects have multiple test suites, and I wanted to avoid any "magic".

That said, we can always adjust this later without it being a breaking change if we want to adjust the approach